.travis.yml

```diff
@@ -1,33 +1,78 @@
-sudo: required
+# File managed by pluginsync
+sudo: true
 language: go
 go:
-- 1.5.4
-- 1.6.2
+- 1.6.3
+- 1.7.1
+addons:
+  apt:
+    packages:
+    - debhelper
+    - cmake
+    - libgeoip-dev
+    - protobuf-compiler
 services:
-- docker
-before_install:
-- go get github.com/tools/godep
-- if [ ! -d $SNAP_PLUGIN_SOURCE ]; then mkdir -p $HOME/gopath/src/github.com/intelsdi-x; ln -s $TRAVIS_BUILD_DIR $SNAP_PLUGIN_SOURCE; fi # CI for forks not from intelsdi-x
+  - docker
 env:
   global:
-    - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/intelsdi-x/snap-plugin-publisher-heka
+  - ORG_PATH=/home/travis/gopath/src/github.com/intelsdi-x
+  - SNAP_PLUGIN_SOURCE=/home/travis/gopath/src/github.com/${TRAVIS_REPO_SLUG}
   matrix:
-    - TEST=unit
-    - TEST=integration
-addons:
-  apt:
-    packages:
-      - debhelper # required for trusty VM
-      - cmake
-      - libgeoip-dev
-      - protobuf-compiler
+  - TEST_TYPE=small
+  - TEST_TYPE=build
```

Comment:
Currently this plugin supports old types of tests (unit and integration) so in this configuration tests are successfully running on travis but it is useless - nothing is tested for now.
This plugin should be adjusted to new names of tests but I think that is should be done in separate pull request. Change of tests names has an impact on several files.
Now, I can just add old types of tests to matrix. 
What do you think?